src/placement/pl_map.c

```diff
@@ -31,6 +31,7 @@
 #include <gurt/hash.h>
 
 extern struct pl_map_ops	ring_map_ops;
+extern struct pl_map_ops	mapless_map_ops;
```

Comment:
(style)  externs should be avoided in .c files

Reply:
Extern required here.